README.md

```diff
@@ -35,7 +48,7 @@ If you use symfony2, you can easily register them in:
 
 - *Recommended way:*
 
-Add to AppKernel
+Add to AppKernel, and make sure it is declared **before** `DoctrineBundle` (if you want disable some).
```

Comment:
would it be possible to make it order independant (by playing with compiler pass priority f.e)?